app/views/includes/dashboard-scripts.html

```diff
@@ -0,0 +1,2 @@
+{% include "includes/datatables-scripts.html" %}
```

Comment:
This will include `jquery.dataTables.min.js` which I assume isn't needed on this page. I'm okay to leave this as I have created an issue for a refactor of the includes directory in general